### PR TITLE
Some info about Baldrmonkey

### PR DESCRIPTION
**Pull request by bnjbvr**

### CODE REVIEW

**Comment by mgaudet** [comment]
I think this got subsumed by #9, so closing. Feel free to reopen if I missed something

**Comment by bnjbvr** [comment]
No, this is something different (Baldr is the Ion backend for wasm, Rabaldr is the wasm baseline).

Jeu 22 août 2019, à 18:32, Matthew Gaudet a écrit :

**Comment by mgaudet** [comment]
Sorry (got lost in my todo list to reopen this): Could you rebase this? Then I'll merge

In hindsight this is why you don't do work on leave, I was insufficiently careful reading the request title. 

Apologies again!

**Comment by bnjbvr** [comment]
Done! Also removed a few trailing whitespaces detected by my editor.

**Comment by bnjbvr** [comment]
ping :)

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-A collection of links to published articles, blog posts, talks, and other important pieces of history of the SpiderMonkey JavaScript engine. 
-
+A collection of links to published articles, blog posts, talks, and other important pieces of history of the SpiderMonkey JavaScript engine.
+A
```
Comment (mgaudet) [inline README.md:2]: Stray `A`?

**Comment by mgaudet** [comment]
Whoops. Merging!

**Comment by mgaudet** [comment]
(I'll fix the A after merging)

**Comment by bnjbvr** [comment]
Cheers!

**Pull request merged**